expense/workflow.go

```diff
@@ -47,13 +48,73 @@ func SampleExpenseWorkflow(ctx workflow.Context, expenseID string) (result strin
 		return "", nil
 	}
 
-	// step 3, request payment to the expense
-	err = workflow.ExecuteActivity(ctx2, PaymentActivity, expenseID).Get(ctx2, nil)
-	if err != nil {
-		logger.Info("Workflow completed with payment failed.", "Error", err)
+	logger.Info(fmt.Sprint("Sending signal for payment for expense ", expenseID))
+
+	activityCtx := workflow.WithActivityOptions(ctx, workflow.ActivityOptions{
+		ScheduleToCloseTimeout: time.Minute * 1,
```

Comment:
I would not use such a short ScheduleToClose timeout as it limits the maximum time for retries. Use StartToClose instead.

Reply:
Also, consider using LocalActivity for such a short call.